From f715a77fbdc11aafff35f066380b34a99401a698 Mon Sep 17 00:00:00 2001
From: Hanno Boeck <ctulhu@users.sourceforge.net>
Date: Mon, 25 Jul 2011 19:39:59 +0100
Subject: [PATCH] Fix compilation with curl 7.21.7 (SF#3376584)

Remove include of curl/types.h, which has been deprecated for a long time.
---
 src/osm-traces.c |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/src/osm-traces.c b/src/osm-traces.c
index 6ef5bb5..f5088ce 100644
--- a/src/osm-traces.c
+++ b/src/osm-traces.c
@@ -27,7 +27,6 @@
 #include <errno.h>
 
 #include <curl/curl.h>
-#include <curl/types.h>
 #include <curl/easy.h>
 
 #include <glib.h>
-- 
1.7.4.1
